src/Service/cart.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+import { API_PATH } from "./constants";
+
+export const cartAPI = {
+  getOrderFromServe: (id) => axios.get(`${API_PATH}/orders?userID=${id}`),
```

Comment:
thường thì đặt tên sẽ theo dạng (find hoặc get, insert, update, remove hoặc delete)
```suggestion
  getOrderById: (id) => axios.get(`${API_PATH}/orders?userID=${id}`),
```
Ngoài ra thì đống này đã nằm trong API thì rõ ràng nó từ server rồi, nên k cần đặt kiểu fromServer làm gì đâu, đặt như này sẽ rõ ràng hơn nhé